ADA/databricks_fundamentals.qmd

```diff
@@ -66,107 +73,118 @@ The storage and computation are separated into different components rather than
+Unlike a SQL database the unity catalog also contains volumes, which are file storage similar to a shared drive. They can be used for storing any type of file.
 
-A schema can contain any number of tables, views and volumes.
+Volumes are stored under a schema within a catalog. Files in here can be accessed and manipulated through code.
 
-### ![](/images/ada-unity-catalog.jpg)
+Examples of files suitable to be stored in a volume include CSVs, JSON and other formats of data files, or supporting files / images for applications you develop through the platform. You can also upload files to a volume through the user interface.
 
-### Workspaces - DataBricks file system (DBFS)
+### Workspaces - Databricks file system (DBFS)
 
-Each user has their own workspace which serves as a personal document and code storage. It contains a user folder which is specific to each person, and any git repositories that you have cloned or created within DataBricks.
+Each user has their own workspace which serves as a personal document and code storage area. It contains a user folder which is only accessible to that user by default, along with any Git repositories that you have cloned or created within Databricks.
 
-Your workspace can be accessed through the 'Catalog' option in the DataBricks sidebar.
+Your workspace can be accessed through the 'Catalog' option in the Databricks sidebar.
 
 ![](/images/ada-workspace-repos.png){fig-align="center"}
 
-Everything in your workspace is only accessible to you unless you share it with other users. When you do share a script or file you can specify whether the person you're sharing it with is able to view/edit/run the file your sharing.
+Everything in your workspace is only accessible to you unless you share it with other users. When you do share a script or file you can specify whether the person you're sharing it with is able to view/edit/run the file you're sharing.
 
 ::: callout-caution
 ## Don't overshare
 
 Sharing code this way can be useful but has it's risks. If you allow other users to edit and run your workbooks it's possible that they can make changes or run it simultaneously resulting in unexpected results.
 
-For collaboration on code it is always preferable to use a github/DevOps repository which each user can clone and work on independently.
+For collaboration on code it is always preferable to use a GitHub/DevOps repository which each user can clone and work on independently.
 :::
 
 ### Repositories for version control
 
-Ideally all code should be managed through a versioned repository on github or Azure DevOps.
+Ideally all code should be managed through a versioned repository on GitHub or Azure DevOps.
 
-To connect databricks to a repository:
+To connect Databricks to a repository:
 
-1.  Create a repository on your chosen platform (github / DevOps) and copy the HTTPS link to the repo.\
+1.  Create a repository on your chosen platform (GitHub / DevOps) and copy the HTTPS link to the repo.\
```

Comment:
We have info on connecting to git on a separate page so can link to that here instead of repeating guidance

Reply:
https://dfe-analytical-services.github.io/analysts-guide/ADA/git_databricks.html